intelmq/bots/parsers/shadowserver/parser.py

```diff
@@ -134,6 +136,8 @@ def parse_line(self, row, report):
         # extra if an add operation failed
         for item in conf.get('optional_fields'):
             intelmqkey, shadowkey = item[:2]
+            if shadowkey not in fields:  # key does not exist in data (not even in the header)
```

Comment:
Hm, but as it's optional it's kind of expected. Would prefer info instead. What do you think?